php/attributes/default.rb

```diff
@@ -6,6 +6,7 @@
 default['php']['extensions']['ini_suffix'] = '-settings'
 
 default['php-apc'] = {}
+default['php-apc']['package_prefix'] = nil
```

Comment:
@gilleyj @till - why is there a special handling for php-apc? Maybe add a quick comment so noone is confused on further refactoring later on.